core/model/modx/moduser.class.php

```diff
@@ -81,6 +85,7 @@ public function setSudo($sudo) {
      */
     public function save($cacheFlag = false) {
         $isNew = $this->isNew();
+        if ($isNew) $this->set('createdon', time());
```

Comment:
@Mark-H You are right. From what I see `modresource.class.php` use: 
```
        if ($this->isNew()) {
            if (!$this->get('createdon')) $this->set('createdon', time());
        }
```
What is your preference?